src/CHANGES.txt

```diff
@@ -10,18 +10,22 @@ NOTE: Please include a reference to any Issues resolved by your changes in the b
 
 RELEASE  VERSION/DATE TO BE FILLED IN LATER
 
+  From Iosif Kurazs:
```

Comment:
You must've missed this bit at the top of this file:
**NOTE: Please add your name below in alphabetical order by last name.**

Reply:
Thank you. I changed.